packages/shared/src/tests/fixtures.ts

```diff
@@ -101,6 +103,15 @@ export const ingressEgressEnvironment = ({
       Polkadot: { DOT: egressFee },
       Ethereum: { ETH: egressFee, FLIP: egressFee, USDC: egressFee },
     },
+    egress_dust_limits: {
+      Ethereum: {
+        ETH: minEgressAmount,
+        USDC: minEgressAmount,
+        FLIP: minEgressAmount,
+      },
+      Polkadot: { DOT: minEgressAmount },
+      Bitcoin: { BTC: '0x258' },
```

Comment:
Any specific reason why the min. egress amount for btc is `0x258`?

Reply:
this is only a test mock, but its set to mainnet for that value because there is something called dust limit for bitcoin

Reply:
basically you cant spend any uxto lower than the dust limit. i think nodes just dont propagate such txs but im not sure how it actually works